sdk/loadtestservice/azure-analytics-loadtestservice/tests/test_smoke.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from testcase import LoadtestserviceTest, LoadtestservicePowerShellPreparer
+
+class LoadtestserviceSmokeTest(LoadtestserviceTest):
+
+    @LoadtestservicePowerShellPreparer()
+    def test_smoke_list_search(self, loadtestservice_endpoint):
+        client = self.create_client(loadtestservice_endpoint)
+        result = client.test.list_load_test_search()
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/24744/files#r901233572